lnbits/core/services.py

```diff
@@ -344,13 +344,11 @@ class PaymentKwargs(TypedDict):
             # payment.ok can be True (paid) or None (pending)!
             logger.debug(f"updating payment {temp_id}")
             async with db.connect() as conn:
+                fee = payment.fee_msat or 0 + service_fee_msat
```

Comment:
Let's make the logic more obvious
```suggestion
                fee = (payment.fee_msat or 0) + service_fee_msat
```